src/pages/Search/Search.js

```diff
@@ -13,14 +13,14 @@ import "./Search.css";
 export default function Search() {
   
   let location = useLocation();
+  const history = useHistory();
```

Comment:
Looks like history should use "let" rather than "const"

See first code block here:
https://dev.to/developer_buddy/react-router-dom-hooks-how-to-use-them-3a93

Reply:
Good catch. Just fixed this